src/website/app/pages/Theming/theming.md

```diff
@@ -132,17 +160,48 @@ const MyButton = createThemedComponent(Button, {
 });
 ```
 
-### `createTheme(baseColor, overrides)`
+### `createTheme(options)`
 
 This function is useful when you want to create a new theme that uses a
 different color scheme or otherwise overrides default values.
 
 **Parameters**
 
-* `baseColor`: Optional.  Default: 'blue'.  Color used to generate theme color
-scheme.  Value must be a valid [Mineral UI color](/color#guidelines-ramps).
-* `overrides`: Optional.  A shallow object of variables to be spread on to the
-default theme.  Useful to override default values.
+* `options`: Optional. An object with the following shape. All properties are optional.
+
+| Option           | Type            | Description                                    |
+|------------------|-----------------|------------------------------------------------|
+| `colors.theme`   | Color or Ramp   | Color used for all "theme" keys in the theme   |
+| `colors.danger`  | Color or Ramp   | Color used for all "danger" keys in the theme  |
+| `colors.success` | Color or Ramp   | Color used for all "success" keys in the theme |
+| `colors.warning` | Color or Ramp   | Color used for all "warning" keys in the theme |
+| `colors.black`   | string          | Color used for black in the theme              |
+| `colors.gray`    | Ramp            | Color used for all "gray" values in the theme  |
+| `colors.white`   | string          | Color used for black in the theme              |
+| `overrides`      | Object          | <key, value> pairs of specific theme overrides |
+
+**Types**
+
+* **Color**: String matching a color ramp name in the Mineral UI [palette](/color#guidelines-ramps)
+* **Ramp**: Object matching this shape:
+
+  ```
+  const myRamp = {
+    [10]: '<color>',
+    [20]: '<color>',
+    [30]: '<color>',
+    [40]: '<color>',
+    [50]: '<color>',
+    [60]: '<color>',
+    [70]: '<color>',
+    [80]: '<color>',
+    [90]: '<color>',
+    [100]: '<color>',
+    inflection: 70
+  };
+  ```
+
+  Note the optional `inflection` key. Mineral UI's color ramps are designed to provide an [accessible level of contrast](/color#guidelines-accessibility). You'll notice that each ramp's text color changes from black to white at the `60` value. If your ramp has a different inflection point, you may use this key to define it.
```

Comment:
I think it would be valuable to make it clear that Mineral expects 10 colors with fairly even steps. Maybe something like: "Note: Mineral expects ramps with 10, evenly distributed color steps. Additionally, Mineral UI's color ramps are designed to provide an [accessible level of contrast](/color#guidelines-accessibility). The provided ramps are designed so that white text color is applied against background colors of 60 and above. Custom palettes with an inflection point at 60 will integrate best into Mineral components, however, if your palette's accessible contrast occurs before or after this step, you may provide your own `inflection` key (meaning this background color will be used for components that implement white text). 

A callout may be appropriate here too?